dev/cnf/oss_dependencies.maven

```diff
@@ -503,12 +503,14 @@ org.eclipse.persistence:org.eclipse.persistence.antlr:2.7.10
 org.eclipse.persistence:org.eclipse.persistence.asm:9.3.0
 org.eclipse.persistence:org.eclipse.persistence.core:2.7.10
 org.eclipse.persistence:org.eclipse.persistence.core:3.0.2
+org.eclipse.persistence:org.eclipse.persistence.core:4.0.0-M3
```

Comment:
Can you use io.openliberty.org.eclipse.persistence:org.eclipse.persistence.core:4.0.0.202d992 in oss_ibm.maven instead?

Reply:
Ahh, did a search for any 4.0 but didn't see it didn't resolve, but absolutely should be able to use that one.